### DISCUSSION TITLE
possible to use juicefs as the disk for mysql database?

### DISCUSSION THREAD

**Discussion by vipsland** [juicedata/juicefs#2660] [Q&A]
/var/lib/mysql location as juicefs mount. is it advisable? 
what kind of problems / caveats shld i be looking out for?

i have a lot of tables and most of them are not accessed. i was thinking 
/var/lib/mysql using b2 backblaze storage (coz wasabi is too slow and b2 is cheaper than s3) and was wonder if this is an advisable setup?

i will of course use the cache more, maybe 100GB of cache

**Comment by davies** [comment]
No, JuiceFS is not recommended to use as the primary storage for databases. It should be good for backup.
